client/curriculum/css/StylingMethods.tsx

```diff
@@ -12,6 +12,7 @@ import Highlight from '~/components/Highlight/Highlight';
 import coverSvg from '~/public/images/lessons/styling-methods__cover.svg';
```

Comment:
Same as in the `BoxModel.tsx`.

Let's keep the file as is, and only modify where we have to replace with `<FormattedText>`.